### ISSUE TITLE
file path to source has hardcoded $HOME

### ISSUE THREAD

**Issue by ckoehler** [Canop/broot#447]
By default, broot adds the following line to my rc file:

`source /Users/me/.config/broot/launcher/bash/br`

That works on my Mac, but on my Linux box my homedir is (obviously) different. How about instead making it: 

`source $HOME/.config/broot/launcher/bash/br`

**Comment by Canop** [comment]
This is very strange and a bug as the source path is built from a written file and a cross-platform lib is used to find the user home.

Are you sure you didn't move a rc file from mac to linux ?

**Comment by ckoehler** [comment]
Oh it’s correct when done on the Mac, and correct when done on Linux, but I share dots across machines and that won’t work. But using $HOME should make it portable, right?

**Comment by Canop** [comment]
> But using $HOME should make it portable, right?

Probably. I don't have a mac to test but I guess you can build something, perhaps using an ad-hoc env variable if $HOME doesn't work.

**Comment by ckoehler** [comment]
Yeah I already fixed it for me, I am saying that maybe that should be the default behavior when starting broot for the first time.

**Comment by Canop** [comment]
OK, I get what you say. I note it but I'd have to check more before I do the move, especially regarding whether I can really be sure to have a $HOME everytime.

**Comment by ckoehler** [comment]
Fair enough, just a suggestion. Thanks!
